### PR TITLE
add dot-env.template

### PR DESCRIPTION
**Pull request by aconchillo**

### CODE REVIEW

File: dot-env.template [hunk 1]
```diff
@@ -0,0 +1,5 @@
+OPENAI_API_KEY=...
+ELEVENLABS_API_KEY=...
+ELEVENLABS_VOICE_ID=...
+DAILY_API_KEY=https://...
```
Comment (Moishe) [inline dot-env.template:4]: This doesn't need the https:// prefix

Comment (aconchillo) [reply dot-env.template:4]: Oooops, copy&paste.

Comment (aconchillo) [reply dot-env.template]: Updated.

**Review by Moishe — APPROVED** [review]
Thanks for doing this!

**Pull request merged**